client/gamescreen.js

```diff
@@ -16,6 +16,10 @@
         var keymap = {};
         var self = this;
 
+        // Current mouse position relative to canvas origin.
+        // For aim vector rendering.
+        var mousePos = {};
```

Comment:
Please use null instead of an empty object.
The error messages will be much more useful.